blank_project/src/main.js

```diff
@@ -78,7 +80,8 @@ function signedInFlow() {
 }
 
 async function setGreeting() {
-    await window.contract.setGreeting({message:'Howdy'});
+    window.nearGreetingIndex = (window.nearGreetingIndex + 1) % 3;
```

Comment:
Can we use `nearGreetings.length` instead of `3`?

Also, I think we can probably get rid of the `window.` in front of all of these – that might make it easier to read, even as we add length with the `nearGreetings.length`

Reply:
done